### PR TITLE
Update Japanese translation (Line 200-249)

### PR DESCRIPTION
**Pull request by t-inu**
/hold for Japanese developer's review.
/assign @atoato88
/cc @yosshy @YuikoTakada @s-kawamura-w664

### CODE REVIEW

File: translations/ja.json [hunk 1]
```diff
@@ -198,55 +198,55 @@
...
+	"Failed runtime": "ランタイムに失敗しました",
+	"Failed to build image": "イメージのビルドに失敗しました",
+	"Failed to cache and load images": "イメージのキャッシュとロードに失敗しました",
+	"Failed to cache binaries": "バイナリーのキャシュに失敗しました",
+	"Failed to cache images": "イメージのキャッシュに失敗しました",
+	"Failed to cache images to tar": "tar へのイメージのキャッシュに失敗しました",
+	"Failed to cache kubectl": "kubectl のキャッシュに失敗しました",
+	"Failed to change permissions for {{.minikube_dir_path}}: {{.error}}": "{{.minikube_dir_path}} に対する権限の変更に失敗しました: {{.error}}",
+	"Failed to check main repository and mirrors for images": "メインリポジトリーのチェックとイメージのミラーに失敗しました",
+	"Failed to configure metallb IP {{.profile}}": "metalib IP {{.profile}} の設定に失敗しました",
```
Comment (s-kawamura-w664) [inline translations/ja.json:239]: Maybe `metallb` is correct (not `metalib`), I think.
metallb = metal load balancer

Comment (yosshy) [reply translations/ja.json:239]: MetalLB is a load-balancer implementation for bare metal Kubernetes clusters, using standard routing protocols.
https://metallb.universe.tf/

**Review by yosshy — COMMENTED** [review]
@t-inu would you check my comments?

File: translations/ja.json (hunk 1 above)
Comment (t-inu) [reply translations/ja.json]: Thank you for pointing out. This is my mistake. Fixed.

File: translations/ja.json [hunk 2]
```diff
@@ -198,55 +198,55 @@
...
-	"Error while setting kubectl current context:  {{.error}}": "",
-	"Error with ssh-add": "",
+	"Error while setting kubectl current context :  {{.error}}": "kubectl の現在のコンテキストの設定中にエラーが発生しました :  {{.error}}",
+	"Error while setting kubectl current context:  {{.error}}": "kubectl の現在のコンテキストの設定中にエラーが発生しました:  {{.error}}",
+	"Error with ssh-add": "ssh-add でエラーが発生しました",
 	"Error writing mount pid": "マウントした pid を書き込み中にエラーが発生しました",
 	"Examples": "例",
-	"Executing \"{{.command}}\" took an unusually long time: {{.duration}}": "",
-	"Existing disk is missing new features ({{.error}}). To upgrade, run 'minikube delete'": "",
+	"Executing \"{{.command}}\" took an unusually long time: {{.duration}}": "「{{.command}}」の実行に非常に長い時間がかかりました: {{.duration}}",
```
[the remark this answers is not in the document]
Comment (t-inu) [reply translations/ja.json]: ここはどちらにしようか迷ったところですが、普通ではないことを強調するため、Suggested changeのように修正しました。

File: translations/ja.json [hunk 3]
```diff
@@ -198,55 +198,55 @@
...
-	"Failed to check main repository and mirrors for images": "",
-	"Failed to configure metallb IP {{.profile}}": "",
-	"Failed to create file": "",
-	"Failed to create runtime": "",
-	"Failed to delete cluster {{.name}}, proceeding with retry anyway.": "クラスタ {{.name}} を削除できませんでしたが、処理を続行します。",
-	"Failed to delete cluster {{.name}}.": "",
-	"Failed to delete cluster: {{.error}}": "",
-	"Failed to delete cluster: {{.error}}__1": "クラスタを削除できませんでした。{{.error}}",
+	"Exiting due to {{.fatal_code}}: {{.fatal_msg}}": "{{.fatal_code}} が原因で終了しています: {{.fatal_msg}}",
+	"Exiting.": "終了しています。",
```
[the remark this answers is not in the document]
Comment (t-inu) [reply translations/ja.json]: そうですね。224行も含め、「終了します」にしました。

File: translations/ja.json [hunk 4]
```diff
@@ -198,55 +198,55 @@
...
+	"Exposed port of the proxyfied dashboard. Set to 0 to pick a random port.": "プロキシー化されたダッシュボードの公開ポート。0 に設定すると、ランダムなポートが選ばれます。",
+	"External Adapter on which external switch will be created if no external switch is found. (hyperv driver only)": "外部スイッチが見つからない場合に、外部スイッチが作成される外部アダプター。(hyperv ドライバーのみ)",
+	"Fail check if container paused": "コンテナーが一時停止しているかどうかのチェックに失敗しました",
```
[the remark this answers is not in the document]
Comment (t-inu) [reply translations/ja.json:229]: Thank you for your comment. In translating this part, I have checked the source code.
https://github.com/kubernetes/minikube/blob/ea443ede8c2e098ff927cd496c99dcb84c71a8f1/cmd/auto-pause/auto-pause.go#L159
This is error message if CheckIfPaused() is failed.
Therefore, I judged that it was the same as any other "Failed" error message.

File: translations/ja.json [hunk 5]
```diff
@@ -198,55 +198,55 @@
...
+	"Failed runtime": "ランタイムに失敗しました",
```
[the remark this answers is not in the document]
Comment (t-inu) [reply translations/ja.json]: Fixed.

**Comment by atoato88** [comment]
I left just one comment.

File: translations/ja.json [hunk 6]
```diff
@@ -198,55 +198,55 @@
...
+	"Fail check if container paused": "コンテナーが一時停止しているかどうかのチェックに失敗しました",
+	"Failed runtime": "ランタイムが失敗しました",
+	"Failed to build image": "イメージのビルドに失敗しました",
+	"Failed to cache and load images": "イメージのキャッシュとロードに失敗しました",
+	"Failed to cache binaries": "バイナリーのキャシュに失敗しました",
+	"Failed to cache images": "イメージのキャッシュに失敗しました",
+	"Failed to cache images to tar": "tar へのイメージのキャッシュに失敗しました",
+	"Failed to cache kubectl": "kubectl のキャッシュに失敗しました",
+	"Failed to change permissions for {{.minikube_dir_path}}: {{.error}}": "{{.minikube_dir_path}} に対する権限の変更に失敗しました: {{.error}}",
+	"Failed to check main repository and mirrors for images": "メインリポジトリーのチェックとイメージのミラーに失敗しました",
```
[the remark this answers is not in the document]
Comment (t-inu) [reply translations/ja.json]: Oh, thanks. I noticed that "mirrors" is not verb, it means mirror repositories.
I fixed as following.
"メインリポジトリーとミラーのイメージのチェックに失敗しました"

[the remark this answers is not in the document]
Comment (atoato88) [reply translations/ja.json]: Thank you to update. I've confirmed that fixed. :smile:

**Comment by atoato88** [comment]
/lgtm
It's OK as Japanese review!

File: translations/ja.json (hunk 1 above)
Comment (s-kawamura-w664) [reply translations/ja.json]: I checked your update too. It's Ok.

**Comment by t-inu** [comment]
@s-kawamura-w664 @yosshy @atoato88 Thank you for your review.
@medyagh Would you please merge it?
/unhold
/assign @medyagh

**Review by spowelljr — APPROVED** [review]

**Comment by t-inu** [comment]
@spowelljr Thank you for merging this PR!

**Pull request merged**